packages/template-retail-react-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/retail-react-app",
-  "version": "1.1.0-dev.0",
```

Comment:
Should the version just be `1.1.0-dev` instead of `1.1.0-dev.1`?

Reply:
Makes sense but I'm not sure how to pass that version to the `bump versions` command.

Reply:
Fixed!